### PR TITLE
assigment 1.2 done

### PR DESCRIPTION
**Pull request by dimonker**
Реализовал линейный классификатор и добился точности 0.202

### CODE REVIEW

File: Rebrik_Dmitry/assignments/assignment1/linear_classifer.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      zeros[target_index] = 1
+
+    # print(sumlog.shape, x_i.shape)
+    loss = sumlog - x_i.T
+    grad = softmax(predictions)
+    grad -= zeros
+
+    if predictions.ndim > 1:
+      grad /= predictions.shape[0]
+    return loss.mean(), grad
```
Comment (toshiks) [inline Rebrik_Dmitry/assignments/assignment1/linear_classifer.py]: Этот код должен использовать написанный тобой softmax и cross_entropy_loss

File: Rebrik_Dmitry/assignments/assignment1/linear_classifer.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    Arguments:
+      probs, np array, shape is either (N) or (batch_size, N) -
+        probabilities for every class
+      target_index: np array of int, shape is (1) or (batch_size) -
+        index of the true class for given sample(s)
+
+    Returns:
+      loss: single value
+    '''
+    return -np.log(probs[target_index])
```
Comment (toshiks) [inline Rebrik_Dmitry/assignments/assignment1/linear_classifer.py]: На выходе этой функции на первый взгляд не будет одного значение. А в требованиях одно значение.

**Comment by toshiks** [comment]
К сожалению, ты не допилил свой код и у тебя просрочен дедлайн
